Makefile

```diff
@@ -1,19 +0,0 @@
-install:
-	poetry install
-
-install_poetry:
-	curl -sSL https://install.python-poetry.org | python -
-	poetry install
-
-tests: install tests_only tests_pre_commit
-
-tests_pre_commit:
-	poetry run pre-commit run --all-files
-
-run_tests: tests
-
-tests_only:
-	poetry run pytest --cov=./ --cov-report=xml --cov-report=html -vv
-
-build_sync:
```

Comment:
Will we be adding an equivalent `tox` command for `build_sync`? Or is there a new proposed flow there

Reply:
aah good point. There is a new proposed flow that I have in mind is to unify the sync and async clients: https://github.com/supabase-community/supabase-py/issues/697

But that's quite a complex change that will need a fair bit of testing + review before it's release ready. For now, it definitely makes sense to keep a corresponding `tox` command for compatibility 👍🏼

Reply:
hey @J0! sorry for the delay, life happened. I've addressed this now -- will mark as resolved.